cli/internal/helm/upgrade.go

```diff
@@ -324,86 +329,86 @@ func (c *UpgradeClient) upgradeRelease(
 }
 
 // loadUpgradeValues loads values for a chart required for running an upgrade.
-func (c *UpgradeClient) loadUpgradeValues(ctx context.Context, conf *config.Config, idFile clusterid.File, chart *chart.Chart,
-) (string, map[string]any, error) {
-	// We need to load all values that can be statically loaded before merging them with the cluster
-	// values. Otherwise the templates are not rendered correctly.
-	k8sVersion, err := versions.NewValidK8sVersion(conf.KubernetesVersion, false)
-	if err != nil {
-		return "", nil, fmt.Errorf("validating k8s version: %s", conf.KubernetesVersion)
-	}
-
-	c.log.Debugf("Checking cluster ID file")
-	clusterName := clusterid.GetClusterName(conf, idFile)
-
-	loader := NewLoader(conf.GetProvider(), k8sVersion, clusterName)
-
-	var values map[string]any
-	var releaseName string
-
-	switch chart.Metadata.Name {
-	case ciliumInfo.chartName:
-		releaseName = ciliumInfo.releaseName
-		var ok bool
-		values, ok = ciliumVals[conf.GetProvider().String()]
-		if !ok {
-			return "", nil, fmt.Errorf("cilium values for csp %q not found", conf.GetProvider().String())
-		}
-	case certManagerInfo.chartName:
-		releaseName = certManagerInfo.releaseName
-		values = loader.loadCertManagerValues()
-	case constellationOperatorsInfo.chartName:
-		releaseName = constellationOperatorsInfo.releaseName
-		values = loader.loadOperatorsValues()
-
-		if err := c.updateCRDs(ctx, chart); err != nil {
-			return "", nil, fmt.Errorf("updating CRDs: %w", err)
-		}
-	case constellationServicesInfo.chartName:
-		releaseName = constellationServicesInfo.releaseName
-		values = loader.loadConstellationServicesValues()
-
-		if err := c.applyMigrations(ctx, releaseName, values, conf); err != nil {
-			return "", nil, fmt.Errorf("applying migrations: %w", err)
-		}
-	case csiInfo.chartName:
-		releaseName = csiInfo.releaseName
-		values = loader.loadCSIValues()
-	case awsLBControllerInfo.chartName:
-		releaseName = awsLBControllerInfo.releaseName
-		values = loader.loadAWSLBControllerValues()
-	default:
-		return "", nil, fmt.Errorf("unknown chart name: %s", chart.Metadata.Name)
-	}
-
-	return releaseName, values, nil
-}
+//func (c *UpgradeClient) loadUpgradeValues(ctx context.Context, conf *config.Config, idFile clusterid.File, chart *chart.Chart,
```

Comment:
Are the functions here not needed anymore? Or is this something unintentionally left over from testing?

Reply:
it's just there temporarily and won't stay :)